pages.zh/common/locate.md

```diff
@@ -13,4 +13,4 @@
 
 - 重新建立文件数据索引数据库。如果要查找最近添加的文件，则需要执行此操作：
 
-`sudo /usr/libexec/locate.updatedb`
```

Comment:
Are you sure this isn't necessary on macOS? This seems pretty intentional, especially because it's the only difference to the Linux page.

Reply:
I just tried `locate/focal,now 4.7.0-1ubuntu1 amd64` on ubuntu, running `sudo updatedb` works fine. Besides, the executable `/usr/libexec/locate.updatedb` does not exist.

Reply:
> Or use 2 examples to specify the difference on `linux` and `osx`?

That would kind of defeat the purpose of our separated platform directories. We shouldn't let the user have to filter out the useful part more than necessary.